sdk/dotnet/Pulumi.Automation/WorkspaceStack.cs

```diff
@@ -376,9 +394,15 @@ public Task RemoveConfigAsync(IEnumerable<string> keys, CancellationToken cancel
                 if (inlineHost != null && inlineHost.TryGetExceptionInfo(out var exceptionInfo))
                     exceptionInfo.Throw();
 
+                if (summaryEvent is null)
+                {
+                    throw new InvalidOperationException("No summary of changes.");
```

Comment:
I think maybe this should be a different exception, since this will bubble to the caller and `InvalidOperationException` kind of implies that the caller provided invalid inputs or that they are otherwise doing something they shouldn't - when really this is a pulumi problem.

Reply:
Since we have the `result` available, we could send `CommandException`  or create a subclass of `CommandException` for this particular case?

Reply:
I think a 1st class exception is the right idea but I don't think it should derive from `CommandException` since those are specific to the CLI writing to the StandardError stream and this case doesn't have that. Could just make a `NoSummaryEventException` or maybe `MissingExpectedEventException` or something